automatic/virtualbox-guest-additions-guest.install/virtualbox-guest-additions-guest.install.nuspec

```diff
@@ -26,7 +26,7 @@ This is a nuspec. It mostly adheres to https://docs.nuget.org/create/Nuspec-Refe
         <!-- version should MATCH as closely as possible with the underlying software -->
         <!-- Is the version a prerelease of a version? https://docs.nuget.org/create/versioning#creating-prerelease-packages -->
         <!-- Note that unstable versions like 0.0.1 can be considered a released version, but it's possible that one can release a 0.0.1-beta before you release a 0.0.1 version. If the version number is final, that is considered a released version and not a prerelease. -->
-        <version>6.1.18</version>
+        <version>6.1.36</version>

```

Comment:
This version already exists in the Chocolatey Community Repository.

Reply:
Really, the latest I can see is 6.1.34